### PR TITLE
Input to list entry

### PR DESCRIPTION
**Pull request by dorosiemers**
Added a global state to get data from the form to the list page.

### CODE REVIEW

File: pages/incidents/index.js [hunk 1]
```diff
@@ -1,41 +1,42 @@
+import React, { useContext } from "react";
 import styled from "styled-components";
+import IncidentForm from "../../components/incidentForm";
+import { useIncidents } from "../../context/listContext";
+
+function IncidentPage() {
+  const incidents = useIncidents().incidents;
+  const setIncidents = useIncidents().setIncidents;
+  console.log(incidents);
```
Comment (Kheirah) [inline pages/incidents/index.js]: Consider removing logs which you no longer need

File: pages/incidents/index.js [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
+  function appendListEntry(time, location, offender, incident) {
+    setIncidents((listEntry) => [
+      ...listEntry,
+      {
+        time,
+        location,
+        offender,
+        incident,
+        id: Math.random().toString().substring(),
```
Comment (Kheirah) [inline pages/incidents/index.js]:
```suggestion
        id: Math.random().toString(),
```
Drop the substring if you don't expect to shorten the string.
While Math.random works well for generating pseudo random numbers, consider using packages like nanoid. It's unconventional to see numbers as IDs and it's more common to see a cryptic set of lowercase & uppercase letters and numbers as IDs, which is what packages like nanoid are great for.

**Review by Kheirah — APPROVED** [review]
Very good! Just some minor details you might want to look into.

File: pages/incidents/index.js (hunk 1 above)
Comment (dorosiemers) [reply pages/incidents/index.js]: I was looking for that one since it always renders twice.

**Pull request merged**